### ISSUE TITLE
"Infinite" Supply Drop Bug

### ISSUE THREAD

**Issue by VonEpic** [WarEmu/WarBugs#10407]
Picked up supplies in Morr's Monastery, and returned to Stonetroll to return them, nothing "removed" the supplies from my character, the other character(s) there did not "drop" theirs either, but were still allowed to drop the supplies indefinitely at the keep flag. Leaving RvR via Book of Binding did the trick (did not test if forced drop from damage worked.) I have video footage of the RvR but it's a lot of footage so let me know if you want to take a look at that.

**Comment by Antalor** [comment]
It happened to me too as well. It was about a 1,5 week ago where I fought on my WP in KV. I've took supplies from Hardwater Falls and provided them to WC Flag. I did it several times when I realized I've got infinite us of it. I've moved to PvE zone but it didn't rested my supplies so my friends said to me to relog to "stop" this bug. Anyway thank you VonEpic for reporting it. I asked on chat if it's a "new" bug and they said it happened to them pretty often so I completly missed with reporting it, so thank you! =) 
Best Regards!
